target/linux/ath79/image/generic.mk

```diff
@@ -337,6 +337,8 @@ define Device/asus_rp-ac66
   IMAGES += factory.bin
   IMAGE/factory.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | \
 	append-rootfs | pad-rootfs
+  IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | \
+	append-rootfs | pad-rootfs | append-metadata
```

Comment:
```IMAGE/sysupgrade.bin``` is already provided by *image/Makefile* and you need to define ```IMAGE_SIZE``` instead to enable sysupgrade image.

see: https://buildbot.openwrt.org/master/images/#/builders/1/builds/497/steps/50/logs/stdio

```WARNING: Image file /builder/shared-workdir/build/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-ath79-generic-asus_rp-ac66-squashfs-sysupgrade.bin is too big: 5701636 > 0```

Reply:
fixed, thank you for clarifying